internal/service/create/create_test.go

```diff
@@ -180,6 +181,11 @@ func (b *createOptionsBuilder) withCredentials(credentials string) *createOption
 	return b
 }
 
+func (b *createOptionsBuilder) withModuleSourcesGitDirectory(moduleSourcesGitDirectory string) *createOptionsBuilder {
```

Comment:
I think it's worth to test this option for returning error when it is invalid, similar to how other options are tested in this file.